<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>620</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(207, 207, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Back_Login.png);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: white</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: transparent;
background-color: rgba(0,0,0,0.3);
border-style: outset;
border-width: 2px;
border-color: white;
border-radius: 10px</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border-width: 0</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>33</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../icons/userwoman.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ln_login">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-width: 1px;
border-color:black;
color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
padding-left: 6px</string>
               </property>
               <property name="placeholderText">
                <string>Digite seu login...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border-width: 0</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>33</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../icons/password.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ln_senha">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>33</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-width: 1px;
border-color:black;
color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
padding-left: 6px</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Digite sua senha...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">background: transparent;
border-width: 0</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="bt_entrar">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	font: 75 12pt &quot;Arial&quot;;
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	border-width: 1px;
	border-color: black
}
QPushButton:pressed{
	font: 75 12pt &quot;Arial&quot;;
	background-color: rgb(0, 0, 0);
	border-radius: 10px;
	color: rgb(255, 255, 255);
	border-width: 1px;
	border-color: white
}</string>
               </property>
               <property name="text">
                <string>ENTRAR</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Back_Login.qrc"/>
 </resources>
 <connections/>
</ui>
